### PR TITLE
2.0.3

### PR DESCRIPTION
**Pull request by Matteo-it**
## What This PR Changes

Check docs-arduino updates: https://github.com/arduino/docs.arduino.cc/pull/1021

### CODE REVIEW

File: scripts/datasheet-rendering/render-datasheets.sh [hunk 1]
```diff
@@ -6,17 +6,30 @@ if [ -n "$CI" ]; then
     # Fix for SSL problem in phantom.js
     # export OPENSSL_CONF=/etc/ssl/
 
+    fc-list :lang=zh
```
Comment (sebromero) [inline scripts/datasheet-rendering/render-datasheets.sh:9]:
```suggestion
```

File: scripts/datasheet-rendering/render-datasheets.sh [hunk 2]
```diff
@@ -6,17 +6,30 @@ if [ -n "$CI" ]; then
...
+
+    apt-get install -y language-pack-zh*
+
+    fc-list :lang=zh
```
Comment (sebromero) [inline scripts/datasheet-rendering/render-datasheets.sh:13]:
```suggestion
```

File: scripts/datasheet-rendering/render-datasheets.sh [hunk 3]
```diff
@@ -6,17 +6,30 @@ if [ -n "$CI" ]; then
     # Fix for SSL problem in phantom.js
     # export OPENSSL_CONF=/etc/ssl/
 
+    fc-list :lang=zh
+
+    apt-get install -y language-pack-zh*
```
Comment (sebromero) [inline scripts/datasheet-rendering/render-datasheets.sh:11]: I'd check if this is even needed.

File: scripts/datasheet-rendering/render-datasheets.sh [hunk 4]
```diff
@@ -6,17 +6,30 @@ if [ -n "$CI" ]; then
...
+    cp -a ./styles/fonts/NotoSansTC*.ttf /usr/share/fonts/truetype/noto-sans-tc
     fc-cache /usr/share/fonts
 
     echo "Font info:"
     fc-list | grep "OpenSans"
     fc-list | grep "RobotoMono"
+    fc-list | grep "NotoSansSC"
+    fc-list | grep "NotoSansTC"
 
+    # apt-get install fonts-wqy-microhei ttf-wqy-microhei fonts-wqy-zenhei ttf-wqy-zenhei
```
Comment (sebromero) [inline scripts/datasheet-rendering/render-datasheets.sh:32]:
```suggestion
```

**Review by sebromero — CHANGES_REQUESTED** [review]
Please see comments

File: scripts/datasheet-rendering/render-datasheets.sh (hunk 3 above)
Comment (Matteo-it) [reply scripts/datasheet-rendering/render-datasheets.sh]: It seems to work without these lines, thank you!

**Review by CloruroDiSodio — APPROVED** [review]

**Pull request merged**